s3_upload/s3_upload.py

```diff
@@ -288,15 +292,56 @@ def monitor_directories_for_upload(config, dry_run):
             failed_files=failed_upload,
         )
 
+        if log_data["uploaded"]:
```

Comment:
I may be missing something here or thinking of something eles, but what is `log_data["uploaded"]`? Is it not `log_data["completed"]`?